src/Options/Applicative/Types.hs

```diff
@@ -109,25 +109,26 @@ data Backtracking
 
 -- | Global preferences for a top-level 'Parser'.
 data ParserPrefs = ParserPrefs
-  { prefMultiSuffix :: String     -- ^ metavar suffix for multiple options
-  , prefDisambiguate :: Bool      -- ^ automatically disambiguate abbreviations
-                                  -- (default: False)
-  , prefShowHelpOnError :: Bool   -- ^ always show help text on parse errors
-                                  -- (default: False)
-  , prefShowHelpOnEmpty :: Bool   -- ^ show the help text for a command or subcommand
-                                  -- if it fails with no input (default: False)
-  , prefBacktrack :: Backtracking -- ^ backtrack to parent parser when a
-                                  -- subcommand fails (default: Backtrack)
-  , prefColumns :: Int            -- ^ number of columns in the terminal, used to
-                                  -- format the help page (default: 80)
-  , prefHelpLongEquals :: Bool    -- ^ when displaying long names in usage and help,
-                                  -- use an '=' sign for long names, rather than a
-                                  -- single space (default: False)
-  , prefHelpShowGlobal :: Bool    -- ^ when displaying subparsers' usage help,
-                                  -- show parent options under a "global options"
-                                  -- section (default: True)
-  , prefTabulateFill ::Int       -- ^ Indentation width for tables
-  } deriving (Eq, Show)
+  { prefMultiSuffix :: String                     -- ^ metavar suffix for multiple options
+  , prefDisambiguate :: Bool                      -- ^ automatically disambiguate abbreviations
```

Comment:
How about, instead of increading the indentation, put the haddock on a new line, e.g.
```haskell
  ...
  , prefDisambiguate :: Bool
      -- ^ automatically disambiguate abbreviations (default: 'False')
  ...
  , foo :: Bar
      -- ^ haddocumentation for foo
  ... 
```

Reply:
Done!